docs/zh/week13/13_1.md

```diff
@@ -0,0 +1,377 @@
+---
+lang-ref: ch.13-1
+title: Graph Convolutional Networks I
```

Comment:
`lang: zh` is missing.

Reply:
> `lang: zh` is missing.

It seems I can't edit the file now. The 'edit' button directs me to 404.

Reply:
Now it's correct. I think you've managed!